src/DynamoCore/Core/Threading/UpdateRenderPackageAsyncTask.cs

```diff
@@ -255,19 +259,27 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, ref List<str
                     // color of 0,0,0,255 (Black), we adjust the color components here.
                     if (graphicItem is Curve || graphicItem is Surface || graphicItem is Solid || graphicItem is Point)
                     {
+                        var newLineColors = new List<byte>();
                         for (var i = 0; i < package.LineStripVertexColors.Count; i += 4)
                         {
```

Comment:
Being new to the code, I'm a bit surprised by this.  We don't really need per-vertex coloring for this, can't we just tell the shader this directly?